src/macaron/repo_finder/commit_finder.py

```diff
@@ -114,10 +114,10 @@ class AbstractPurlType(Enum):
     UNSUPPORTED = (2,)
 
 
-def find_commit(git_obj: Git, purl: PackageURL) -> tuple[str, str]:
+def find_commit(git_obj: Git, purl: PackageURL) -> str:
```

Comment:
It's better to change the return type to `str | None` and return `None` instead of empty string to catch type errors.